Incorrect directory

I am getting a build error 

> ./node_modules/formsy-material-ui/lib/FormsyAutoComplete.js
> Module not found: Can't resolve 'formsy-react/src/utils' in '/app/front/node_modules/formsy-material-ui/lib'

formsy-material-ui/lib/FormsyAutoComplete.js
33. var _utils = require('formsy-react/src/utils');

should the correct path be changed to

 './utils'

or should this module be deleted now that MU has moved away from autocomplete?

I have faced the same error too. 

> ./node_modules/formsy-material-ui/lib/FormsyAutoComplete.js
Module not found: Can't resolve 'formsy-react/src/utils' in '/app/node_modules/formsy-material-ui/lib'


```
"dependencies": {
    "axios": "^0.18.0",
    "create-react-class": "^15.6.3",
    "formsy-material-ui": "^0.6.3",
    "formsy-react": "^1.1.4",
    "inline-style-prefixer": "^4.0.0",
    "lint": "^1.1.2",
    "material-ui": "^1.0.0-beta.34",
    "material-ui-icons": "^1.0.0-beta.17",
    "prop-types": "^15.6.1",
    "react": "^16.2.0",
    "react-dom": "^16.2.0",
    "react-router-dom": "^4.2.2",
    "react-scripts": "1.1.1",
    "sockjs-client": "^1.1.4"
  },
```

--- a/src/FormsyAutoComplete.tsx
+++ b/src/FormsyAutoComplete.tsx
@@ -4,7 +4,8 @@ import PropTypes from 'prop-types';
 import keycode from 'keycode';
 import Formsy from 'formsy-react';
 import AutoComplete from 'material-ui/AutoComplete';
-import { setMuiComponentAndMaybeFocus } from 'formsy-react/src/utils';
+
+import { setMuiComponentAndMaybeFocus } from 'utils';
 
 const FormsyAutoComplete = createClass<any, any>({
   propTypes: {

